### ISSUE TITLE
Project Maintenance

### ISSUE THREAD

**Issue by csokolove** [open-notify/Open-Notify-API#40]
As mentioned in #21 its apparent that maintenance on this project has sort of halted, based on both the status from @natronics & the last commit date. I'd be willing to take on some of the maintenance and project maintenance for the Open Notify project, to both ensure bugs are taken care of & features are being updated at a consistent rate.

If others would be willing to contribute to this, please do let me know to form a stronger developer backing behind this project's lifespan.

**Comment by sseshan7** [comment]
Hello @csokolove ! Thank you for taking the initiative for maintaining this project! I am definitely interested in contributing -- performing code reviews, fixing bugs, and growing the project.

Would love to help you keep this project going.

**Comment by csokolove** [comment]
I've started looking into the project and trying to figure out how to improve upon it.

Does anyone happen to know where the data for `astros.json` is coming from? The website says he manually updates it, and all I can find is hard-coded names in the code. I may be missing something, but the data in this code doesn't match up with the current API?

**Comment by sseshan7** [comment]
I agree, the only thing I see are hard coded names. The live API is outputting different data than what is currently specified in the repository.

I'm not sure where the data is coming from today. Would love if @natronics could comment on that.

A quick search for other API's and data sources yielded the following data sources which can be potentially used to populate `astros.json`:

- [https://www.howmanypeopleareinspacerightnow.com/peopleinspace.json](https://www.howmanypeopleareinspacerightnow.com/peopleinspace.json). This endpoint has errors, it reports the 3 astronauts on Shenzhou 13 as being on the ISS. But the names, and the number of astronauts are correct.
- [https://thespacedevs.com/llapi](https://thespacedevs.com/llapi) Launch Library 2. This is a collection of endpoints that can be used to determine the names and number of astronauts in space. This source code is not publicly available.
- [https://spacelaunch-sandbox.mxapps.io/](https://spacelaunch-sandbox.mxapps.io/) An example site that uses the Launch Library 2 API's to report astros currently onboard the ISS.
